features/largeVideo/components/LargeVideo.js

```diff
@@ -34,23 +50,32 @@ class LargeVideo extends Component {
  *
  * @param {Object} state - Redux state.
  * @returns {{
+ *      avatar: string,
  *      videoTrack: Track
  * }}
  */
-const mapStateToProps = state => ({ // eslint-disable-line arrow-body-style
-    videoTrack:
-        getTrackByMediaTypeAndParticipant(
+const mapStateToProps = state => {
```

Comment:
I wrote about this yesterday but I'm writing it again because I still believe it can be improved: `const mapStateToProps = state => ` is easier to comprehend as `function mapStateToProps(state) {`. (If you embark on fixing this, please create a separate, self-contained PR that deals with the replacement of such cases of an anonymous function assigned to a variable only.)